src/Sulu/Bundle/CustomUrlBundle/EventListener/CustomUrlSerializeEventSubscriber.php

```diff
@@ -85,12 +85,20 @@ public function onPostSerialize(ObjectEvent $event)
             $customUrlProperty
         );
 
-        $creatorFullName = $this->userManager->getFullNameByUserId($customUrl->getCreator());
+        $creatorFullName = null;
+        if ($creator = $customUrl->getCreator()) {
```

Comment:
We use yoda conditions to avoid accidently assignements, but if you insist I can change it.

Reply:
Depending on the situation yoda conditions won't help:

```php
if ($a = $b) {}
```

And since we haven't done this yet (at least not often) I would prefer not to start with it now.